deployment/bicep/main.bicep

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+targetScope = 'subscription'
+
+@description('The common name for this application')
+param applicationName string
+
+@description('Location of resources')
+@allowed([
+  'westeurope'
+  'northeurope'
+  'westus'
+  'swedencentral'
```

Comment:
:rocket:

Reply:
Is this list restricted because not all services are available in other regions? or is this just a starter for 10 list, we can add more regions later?

Reply:
I only used some as an example. We can of course add (and maybe remove) regions